docs/velox-backend-support-progress.md

```diff
@@ -104,8 +104,7 @@ Gluten supports 164 functions. (Draw to right to see all data types)
 | >                             | gt                     | greaterthan           | S      |                        | S       | S    | S     | S   | S    | S     | S      |      |           | S      |         |      |        |          |       |      |        |      |
 | >=                            | gte                    | greaterthanorequal    | S      |                        | S       | S    | S     | S   | S    | S     | S      |      |           | S      |         |      |        |          |       |      |        |      |
 | ^                             | bitwise_xor            |                       | S      |                        |         |      | S     | S   | S    |       |        |      |           |        |         |      |        |          |       |      |        |      |
-| \                             |                        | bitwise_or            | bitwise_or            | S      |                        |         |      | S     | S   | S    |       |        |      |           |        |         |      |        |          |       |      |        |      |
-| \                             | \                      |                       |                        |                       |        |                        |         |      |       |     |      |       |        |      |           |        |         |      |        |          |       |      |        |      |
```

Comment:
This line may need to be kept for `||` function with some changes. See [link](https://spark.apache.org/docs/latest/api/sql/index.html#_18).

Reply:
Thanks for your info, i will modify this based on the above info.